### PR TITLE
Mods with exclusively vulnerable versions are not added to the list

### PR DESCRIPTION
**Pull request by zkxs**
I am exceptionally rusty at python, so please make sure I'm not doing anything stupid @ljoonal

### CODE REVIEW

File: gen_readme.py [hunk 1]
```diff
@@ -39,6 +39,25 @@
...
         README += f"[{mod['name']}]({mod['sourceLocation']})"
 
+        # mod must have versions
+        if mod["versions"] is None or len(mod["versions"]) == 0:
+            continue
+
+        # mod must have a non-vulnerable version
+        all_vulnerable = True
+        for key, version in mod["versions"]:
+            if version.flags is not None:
```
Comment (ljoonal) [inline gen_readme.py:49]: Wouldn't this have a mod without any flags ever being set be defined as having all vulnerable?

Comment (zkxs) [reply gen_readme.py]: Whoops... fixed in  03ac8e90

**Review by ljoonal — APPROVED** [review]
My python is rusty too, but assuming that continue works like I imagine it would then seems good.
seal

**Comment by zkxs** [comment]
lol lets find out... I'm just assuming too :P

**Comment by zkxs** [comment]
Aaaand I can't Python. https://github.com/neos-modding-group/neos-mod-manifest/runs/6138792442?check_suite_focus=true

**Pull request merged**